modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/configurators/100_liferay_image_setup.sh.tpl

```diff
@@ -18,17 +18,29 @@ function copy_and_remove_scripts {
 	echo "[LIFERAY] Copying /home/liferay/configs/${LIFERAY_WORKSPACE_ENVIRONMENT}/scripts files:"
 	echo ""
 
+	if [ -d "/home/liferay/configs/${LIFERAY_WORKSPACE_ENVIRONMENT}/scripts)" ] && [ "$(ls -A /home/liferay/configs/${LIFERAY_WORKSPACE_ENVIRONMENT}/scripts)" ]; then
```

Comment:
Can you only check these two asserts once instead of 3 times and then indent all the logic below it?